<ui version="4.0" >
 <class>MTXMLServerClass</class>
 <widget class="QMainWindow" name="MTXMLServerClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MTXMLServer</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <widget class="QPushButton" name="ButtonFileName" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>10</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Open XMLFile</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditSearch" >
    <property name="geometry" >
     <rect>
      <x>330</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>90</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Tag</string>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Tag Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAddElement" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>340</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Add Root</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDelete" >
    <property name="geometry" >
     <rect>
      <x>500</x>
      <y>60</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAddInListElement" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>340</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Add in RightList</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textSrc" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>80</y>
      <width>221</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textSearch" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>180</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSearchNextTagToTag" >
    <property name="geometry" >
     <rect>
      <x>400</x>
      <y>150</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonCreateFileName" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>40</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Create XMLFile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDelete_2" >
    <property name="geometry" >
     <rect>
      <x>510</x>
      <y>130</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditSearchTag" >
    <property name="geometry" >
     <rect>
      <x>330</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSearch" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>FindFirst</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSearchTag" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>120</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>FindFirst</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSearchNextTag" >
    <property name="geometry" >
     <rect>
      <x>380</x>
      <y>120</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSearchTagToTag" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>150</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>FindFirstTag</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSearchNext" >
    <property name="geometry" >
     <rect>
      <x>380</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Next</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
